ui/src/app/cron-workflows/cron-workflow-list.tsx

```diff
@@ -156,7 +156,7 @@ export function CronWorkflowList({match, location, history}: RouteComponentProps
                                         <div className='columns small-2'>{w.metadata.namespace}</div>
                                         <div className='columns small-1'>{w.spec.timezone}</div>
                                         <div className='columns small-1'>
-                                            {w.spec.schedule != ''
+                                            {(w.spec.schedule ?? '') != ''
```

Comment:
```suggestion
                                            {w.spec.schedule
```

you can just use [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) for all these